DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -98,13 +99,16 @@ public void Update_login_count_should_not_increment_login_count_if_session_time_
             var loginTime = new DateTime(2011, 9, 23);
             var expectedLoginCount = courseContentTestHelper.GetLoginCount(progressId);
 

```

Comment:
All the update tests should be in transaction scope, also the ones were we don't expect them to change anything. Because if those tests fail then they will have changed the db, and we still don't want that change to persist 🙂